src/components/SVG/Plus.js

```diff
@@ -1,4 +1,4 @@
-const PlusIcon = ({ options }) => {
+const PlusIcon = ({ options, setDateFilter }) => {
   const { width, height } = options;
```

Comment:
and why not stylize through the class? why is it necessary to pass parameters to svg?